packages/vscode-extension/src/controls/SampleGallery.tsx

```diff
@@ -60,6 +60,7 @@ const imageMapping: { [p: string]: any } = {
   "deep-linking-hello-world-tab-without-sso-M365": DeepLinking,
   "team-central-dashboard": DashboardTab,
   "hello-world-teams-tab-and-outlook-add-in": TeamsTabAndOutlookAddin,
+  "outlook-add-in-set-signature": TeamsTabAndOutlookAddin,
```

Comment:
@davidchesnut Please change the image for your sample here.  An example of adding image and importing it is https://github.com/OfficeDev/TeamsFx/pull/7934/files#diff-ee291c5a28fb9482725ace75e8f5fde2dca3ed4a293186ff7c89814da5ec8773.